bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/BusEnergyMeter.xml

```diff
@@ -31,6 +33,12 @@
 				<description>A "5" must be added before the energy meter address: 5N with N=[1-255]. Example energy meter "42" -->
 					where="542"</description>
 			</parameter>
+
+			<parameter name="energyRefreshPeriod" type="integer" min="30" max="1440" required="true">
```

Comment:
No need to defined it as required as you provide a default value (30 minutes).

Reply:
So I  have to remove the property _min="30"_, right?

Reply:
No, min will justprevent the user to enter a value lower than 30.
If this is what you want, keep it.

What you have to remove is `required="true"`

Reply:
Ok I got it! There no need of _required=true_ because a default value is foreseen!

Reply:
Done